v201710/sandbox_test.go

```diff
@@ -933,3 +933,40 @@ func TestRateError(t *testing.T) {
 	wg.Wait()
 
 }
+
+func Testrequest(t *testing.T) {
+	config := getTestConfig()
+	campaigns, _, err := NewCampaignService(&config.Auth).Get(Selector{
+		Fields: []string{"Id", "Name", "CampaignId"},
+	})
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	campaignId := campaigns[0].Id
+	adGroups, err := NewAdGroupService(&config.Auth).Mutate(
```

Comment:
Does this call actually return an error? I think the point of this test should be to do something that will _definitely_ return an error and then inspect the `err` to make sure it isn't an empty string

Reply:
it might be easier to make a mock http client which returns the actual response you want to make a regression test with

Reply:
@davewalk : It definitely returns an error here, adServingOptimizationStatus is not set. It is worth looking into @slofurno suggestion here as well